tests/small_sent_cnn.ini

```diff
@@ -0,0 +1,96 @@
+;; Small training test
+
+[main]
+name="translation"
+tf_manager=<tf_manager>
+output="tests/outputs/small"
```

Comment:
Ty testy nemůžou mít stejnou output directory. Pak na tom Travisu na sebe chtějí navazovat a spadne to, protože mají navzájem jiné proměnné.